docs/database/supplementary-guidelines/migration-guide.md

```diff
@@ -0,0 +1,249 @@
+
+References:
+
+- [Metric `db.client.connections.usage` v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-metrics.md#metric-dbclientconnectionsusage)
+- [Metric `db.client.connection.count` v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/database-metrics.md#metric-dbclientconnectioncount)
+
+#### Database client connection idle max
+
+Metric changes:
+
+- **Name**: `db.client.connections.idle.max` &rarr; `db.client.connection.idle.max`
+- **Attributes**: see table below
+
+<!-- prettier-ignore-start -->
+| Attribute change                                    | Comments |
+|-----------------------------------------------------|----------|
+| `pool.name` &rarr; `db.client.connection.pool.name` |          |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Metric `db.client.connections.idle.max` v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-metrics.md#metric-dbclientconnectionsidlemax)
+- [Metric `db.client.connection.idle.max` v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/database-metrics.md#metric-dbclientconnectionidlemax)
+
+#### Database client connection idle min
+
+Metric changes:
+
+- **Name**: `db.client.connections.idle.min` &rarr; `db.client.connection.idle.min`
+- **Attributes**: see table below
+
+<!-- prettier-ignore-start -->
+| Attribute change                                    | Comments |
+|-----------------------------------------------------|----------|
+| `pool.name` &rarr; `db.client.connection.pool.name` |          |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Metric `db.client.connections.idle.min` v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-metrics.md#metric-dbclientconnectionsidlemin)
+- [Metric `db.client.connection.idle.min` v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/database-metrics.md#metric-dbclientconnectionidlemin)
+
+#### Database client connection max
+
+Metric changes:
+
+- **Name**: `db.client.connections.max` &rarr; `db.client.connection.max`
+- **Attributes**: see table below
+
+<!-- prettier-ignore-start -->
+| Attribute change                                    | Comments |
+|-----------------------------------------------------|----------|
+| `pool.name` &rarr; `db.client.connection.pool.name` |          |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Metric `db.client.connections.max` v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-metrics.md#metric-dbclientconnectionsmax)
+- [Metric `db.client.connection.max` v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/database-metrics.md#metric-dbclientconnectionmax)
+
+#### Database client connection pending requests
```

Comment:
On the PR https://github.com/open-telemetry/semantic-conventions/pull/1290 I changed the description from `cumulative` to `current`. Do you also want to add this change here or is that to much level of details and not necessary here?
Up to you!

Reply:
I saw that was more of a clarification since in general cumulative vs delta is a backend concern